_docs/tasks/ingress.md

```diff
@@ -58,13 +58,20 @@ to configure ingress behavior.
    ```bash
    kubectl get ingress -o wide
    ```
-   
    ```
    NAME      HOSTS     ADDRESS                 PORTS     AGE
    gateway   *         130.211.10.121          80        1d
-   export INGRESS_URL=130.211.10.121:80
    ```
-
+   In this example we would want to `export INGRESS_URL=130.211.10.121:80`, so:
+   ```bash
+   export INGRESS_URL=`kubectl get ingress -o wide | tail -1 | awk '{printf("%s:%s", $3,$4)}'`
+   echo INGRESS_URL=$INGRESS_URL
+   ```
+   which outputs
+   ```
+   INGRESS_URL=130.211.10.121:80
+   ```
```

Comment:
The :80 part was a bug ... see latest version of the task. 

The instructions for the "load balancers are not supported" case has become more complicated, so I'm gone in the opposite direction and made those instructions manual.

I think that if we want to make the instructions to simply call a command that sets the env variable automatically, we should figure it out for all cases.